java/core/psoxy-core.iml

```diff
@@ -42,6 +42,7 @@
     <orderEntry type="library" name="Maven: com.google.http-client:google-http-client:1.43.3" level="project" />
     <orderEntry type="library" name="Maven: org.apache.httpcomponents:httpclient:4.5.14" level="project" />
     <orderEntry type="library" name="Maven: commons-logging:commons-logging:1.2" level="project" />
+    <orderEntry type="library" name="Maven: org.apache.httpcomponents:httpcore:4.4.16" level="project" />
```

Comment:
No idea why intelliJ does that, is not used in any place.

Reply:
remove it?